libraries/standard/http/include/http_client.h

```diff
@@ -215,18 +224,76 @@ typedef enum HTTPStatus
     HTTP_INSUFFICIENT_MEMORY,
 
     /**
-     * @brief Represents all errors not related to user-input or transport I/O, but
-     * errors internal to the implementation of the HTTP client library.
+     * @brief The server sent more headers than the configured
+     * #HTTP_MAX_RESPONSE_HEADERS_SIZE_BYTES.
      *
      * Functions that may return this value:
      * - #HTTPClient_Send
-     * - #HTTPClient_ReadHeader
      */
-    HTTP_INTERNAL_ERROR,
     HTTP_SECURITY_ALERT_RESPONSE_HEADERS_SIZE_LIMIT_EXCEEDED,
-    HTTP_SECURITY_ALERT_PARSER_INVALID_CHARACTER,
-    HTTP_SECURITY_ALERT_INVALID_CONTENT_LENGTH,
-    /* TODO: Add return codes as implementation continues. */
+
+    /**
+     * @brief The server data past a complete response that contained the
```

Comment:
This description is not clear to me. Does it mean that the server response contained data past the "Connection: close" header that is not expected?

Reply:
```
     * @brief A response contained the "Connection: close" header, but there
     * was more data at the end of the complete message.
```